.gitignore

```diff
@@ -77,12 +77,20 @@ wp-content/debug.log
 /wp-content/index.php
 /wp-content/sunrise.php
 /wp-content/Icon
+/wp-content/ewww/*
 /wp-content/plugins/*
 /wp-content/themes/*
+wp-content/hub-acq-man/*
+wp-content/hub-connector/*
+wp-content/old-hub-plugins/*
 
 # do version-control the theme we care about
 !/wp-content/themes/citylimits
 
+# ignore generated files
+wp-content/themes/citylimits/package-lock.json
+wp-content/themes/citylimits/style.css
```

Comment:
The root `style.css` in a child theme must be tracked in version control; it's how WordPress knows details about the theme.

package-lock.json should be committed herein as well.

Reply:
Ah, I didn't know that package-lock should be tracked, but now I do. But style.css gets generated from less/style.less. Does it need to be tracked? (Splitting hairs, I know.)

Reply:
Yes, it does, because we want this repo to be deployable from a `git push` if Flywheel ever implements something like WPE has.

Reply:
OK, added style.css again!

Reply:
👍